packages/fast-components-styles-msft/src/utilities/color/neutral-focus.spec.ts

```diff
@@ -11,6 +12,8 @@ describe("neutralFocus", (): void => {
     });
 
     test("should operate on default design system if no design system is supplied", (): void => {
-        expect(neutralFocus({} as DesignSystem)).toBe("#101010");
+        expect(contrast(neutralFocus({} as DesignSystem), "#FFF")).toBeGreaterThanOrEqual(
+            3.5
```

Comment:
why was this test updated, contrast does not appear to have been changed

Reply:
This change probably came from the future. With the focus value no longer fixed, even on a white background, this doesn’t work with the palette change that’s coming in soon. I figured this was more resilient of a test anyway.